redash/controllers.py

```diff
@@ -488,6 +488,22 @@ def csv_response(query_result):
         headers.update(cache_headers)
         return make_response(s.getvalue(), 200, headers)
 
+    @staticmethod
+    def add_access_control_allow_origin_header(headers={}):
```

Comment:
I'm not sure about Flask's life cycle, but if controllers are reused, the default headers will be reused, aren't they?